### PR TITLE
Bump the library-go and bump kube to 1.12.5

### PR DESCRIPTION
**Pull request by mfojtik**
In reaction, force to use protobuf client in operator.

/cc @sttts 
/cc @ravisantoshgudimetla

### CODE REVIEW

File: pkg/operator/resourcesynccontroller/resourcesynccontroller.go [hunk 1]
```diff
@@ -18,7 +18,8 @@ func NewResourceSyncController(
 	resourceSyncController := resourcesynccontroller.NewResourceSyncController(
 		operatorConfigClient,
 		kubeInformersForNamespaces,
-		kubeClient,
+		kubeClient.CoreV1(),
+		kubeClient.CoreV1(),
```
Comment (sttts) [inline pkg/operator/resourcesynccontroller/resourcesynccontroller.go]: this is what?

Comment (mfojtik) [reply pkg/operator/resourcesynccontroller/resourcesynccontroller.go]: switch to Getters, it is good change

**Comment by deads2k** [comment]
known flakes.  We're going to be in here adjusting API.  I'm going for this.

**Pull request merged**